bridges/NextInpactBridge.php

```diff
@@ -6,29 +6,105 @@ class NextInpactBridge extends FeedExpander {
 	const URI = 'https://www.nextinpact.com/';
 	const DESCRIPTION = 'Returns the newest articles.';
 
+	const PARAMETERS = array( array(
+		'feed' => array(
+			'name' => 'Feed',
+			'type' => 'list',
+			'values' => array(
+				'Tous nos articles' => 'news',
+				'Nos contenus en accès libre' => 'acces-libre',
+				'Blog' => 'blog',
+				'Bons plans' => 'bonsplans'
+			)
+		),
+		'filter_premium' => array(
+			'name' => 'Premium',
+			'type' => 'list',
+			'values' => array(
+				'No filter' => '0',
+				'Hide Premium' => '1',
+				'Only Premium' => '2'
+			)
+		),
+		'filter_brief' => array(
+			'name' => 'Brief',
+			'type' => 'list',
+			'values' => array(
+				'No filter' => '0',
+				'Hide Brief' => '1',
+				'Only Brief' => '2'
+			)
+		)
+	));
+
 	public function collectData(){
-		$this->collectExpandableDatas(self::URI . 'rss/news.xml', 10);
+		$feed = $this->getInput('feed');
+		if (empty($feed))
+			$feed = 'news';
```

Comment:
Instead of checking for an empty feed you can just specify the default value of the parameter itself and make it required:

```PHP
		'feed' => array(
			'name' => 'Feed',
			'type' => 'list',
			'required' => true,
			'values' => array(
				'Tous nos articles' => 'news',
				'Nos contenus en accès libre' => 'acces-libre',
				'Blog' => 'blog',
				'Bons plans' => 'bonsplans'
			),
			'defaultValue' => 'news'
		),
```

That way the function can be reduced to:

```PHP
	public function collectData(){
		$this->collectExpandableDatas(self::URI . 'rss/' . $this->getInput('feed') . '.xml');
	}
```

As bonus you receive an error message if the parameter is omitted:

![image](https://user-images.githubusercontent.com/5776685/45256913-8c4e9e00-b39d-11e8-9455-ce925053bc83.png)

Reply:
The parameter is implemented this way for backward compatibility, because initially, this bridge did not have the `feed` parameter. If we make the field mandatory, a user upgrading RSS-Bridge will have to notice that and re-create a valid URL for the `NextInpactBridge`. Is it OK to keep that for compatibility?

Reply:
I see, that makes sense. With that in mind it makes sense not to use required parameters, so let's keep it how it is :+1: